modules/virt-about-containerdisks-for-vms.adoc

```diff
@@ -16,12 +16,18 @@ to create large numbers of virtual machine clones.
 When a virtual machine with a containerDisk volume starts, the container
 is pulled from the registry and hosted on the node that is hosting the virtual machine.
 
-[IMPORTANT]
+[NOTE]
 ====
 A containerDisk volume is ephemeral. It is discarded when
 the virtual machine is stopped, restarted, or deleted. A containerDisk
 is useful for read-only filesystems such as CD-ROMs or for disposable
-virtual machines. It is not recommended for read-write filesystems as
-the data is temporarily written to local storage on the hosting node, which
-impedes live migration of the virtual machine in the case of node maintenance.
+virtual machines.
+====
+
+[IMPORTANT]
+====
+Container disks are not recommended for read-write filesystems as
+the data is temporarily written to local storage on the hosting node. This slows
+live migration of the virtual machine, such as in the case of node maintenance,
+because the entire disk must be migrated to the destination node.
```

Comment:
Well, it's not the entire disk, but only the layer of diffs from the base images shipped in the container.

Reply:
We should also add that in the case of a powerouttage the data will be lost.

Reply:
- Have s/entire disk/data/ for accuracy. I don't think we need to be more specific than that here, but I certainly appreciate the clarification.
- Added "Additionally, all data is lost if the node loses power or otherwise shuts down unexpectedly."